participation.html

```diff
@@ -13,7 +13,13 @@
     <div class="banner white">
       <div class="container">
         <h3 class="size8">Scout/Parent Participation</h3>
-        <p>Scout and parent participation are crucial to the success of Troop 370.  Learning, leadership, and advancement opportunities abound for Scouts.<br><br>For parents, there are many ways in which you can assist through volunteer efforts.  There is no greater joy than joining your Scout through their Troop 370 experience, and working to support the Troop "behind the scenes".  If you would like to see how you can help, please contact the Troop, attend a meeting, or check out the "What Can I Do?" link in the password protected members section of the site.  Meetings are held most Wednesday evenings from 7:30 - 9:00PM in the St. James UMC activities building.<br><br>Using the drop-down menu under "Scout/Parent Info" above, you will find forms, helpful information, and other common documents that you will find helpful during your journey through Scouts at Troop 370.</p>
+        <p>Scout and parent participation are crucial to the success of Troop 370.  Learning, leadership, and advancement opportunities abound for Scouts.</p>
+        <p>At Troop 370, there are monthly weekend campouts that the troop takes. Also, every summer, the Troop visits Woodruff Scout Reservation in Blairsville, GA. Recently, they have also been visiting the Summit Bechtel Reserve in West Virginia as a secondary summer camp. The Troop has a multitude of high adventure trips as well. They visit all high adventure bases (more info on high adventure <href> here). The troop also has many service projects open to scout participation.</p>
+        <p>Troop 370 is a boy-led troop, meaning that while the adult scouters perform administrative duties, the scouts plan activities and campouts. Scouts have the opportunity to take up leadership positions in the troop. While leadership positions are required for advancement, the level of the position is mostly optional. There are a multitude of leadership positions available for scouts to hold. Other than the required leadership for advancement, the level of scout participation for leadership is up to him.</p>
+        <p>Advancement moves at the pace of the scout, so he can move as quickly or as slowly as he wants or as time permits. The upper ranks have minimum time requirements and require leadership positions for 4-6 months, so progress is somewhat spaced out, but there is enough to do during the time in between.</p>
```

Comment:
```suggestion
        <p>Advancement moves at the pace of the scout, so he can move as quickly or as slowly as he wants or as time permits. Troop instructors and assistant scoutmasters are happy to help any scout with the advancement requirements. </p>
```

That last sentence I removed is somewhat confusing.
I added a sentence emphasizing the openness towards helping fellow scouts advance.